docs/deep-dives/mobile/first-run-experiments.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: mobile-first-run-experiments
+title: First Run Experiments
+slug: /mobile-first-run-experiments
+---
+
+First run experiments are incredibly important to the experimentation ecosystem, but their functionality comes with some key differences from standard experiments. It is _highly_ advised that experiment owners read through this documentation before writing first run experiments.
```

Comment:
I would *strongly* suggest that you spell `First Run Experiments` with consistent capitalization, as so we can refer to an experiment as a "First Run Experiment", and everyone knows what it is.